### ISSUE TITLE
Question about loss 

### ISSUE THREAD

**Issue by Kyrie-Chen** [gurkirt/FPN.pytorch1.0#6]
**line368 in train.py**
if loc_loss>1000:
      lline = '\n\n\n We got faulty LOCATION loss {} {} \n\n\n'.format(loc_loss, conf_loss)
      log_file.write(lline)
      print(lline)
      loc_loss = 20.0

Why make  _loc_loss_ become 20.0 when _loc_loss>1000_?

**Comment by gurkirt** [comment]
I am not really sure what could be the cause. I also got similar error sometimes, however, if I restarted the training it would train ok.

**Comment by Kyrie-Chen** [comment]
In my opinion, according to OHEM, the weight of the sample whose loss is too large should be increased. But in this code, when loss is larger than 1000, it becomes 20. So it makes me confused.
Or it is not used for OHEM? Then where is the code for OHEM?

**Comment by gurkirt** [comment]
here , https://github.com/gurkirt/FPN.pytorch1.0/blob/master/modules/detection_loss.py
